masonite/view.py

```diff
@@ -51,3 +75,120 @@ def extend(self):
     def share(self, dictionary):
         self.dictionary.update(dictionary)
         return self
+
+    def cache_for(self, time=None, type=None):
+        """
+        Set time and type for cache
+        """
+
+        self.cache_time = time
+        self.cache_type = type
+
+        if self.__is_expired_cache():
+            self.__create_cache_template(self.template)
+        return self
+
+    # --------
+    # Privates
+    # --------
+
+    @property
+    def __get_path_cache(self):
+        """
+        Get path location disk config
+        """
+
+        cache_config = self.container.make('CacheConfig')
+        path = cache_config.DRIVERS['disk']['location'] + "/"
+        return path
+
+    def __create_cache_template(self, template):
+        """
+        Save in the cache the template
+        """
+
+        self.container.make('Cache').store(
+            template + ":" + str(time.time()) + '.html',
+            self.rendered_template
+        )
+
+    def __cached_template_exists(self):
+        """
+        Check if the cache template exists
+        """
+
+        path = self.__get_path_cache
```

Comment:
since we are removing the `@property` then that means that all instances of `self.__get_path_cache` should be `self.__get_path_cache()`. Make sure they are a callable

Reply:
Yes!